### DISCUSSION TITLE
Great Progess 1-15-2022

### DISCUSSION THREAD

**Discussion by coverbeek** [frc5567/2023-Robot#217] [Team Posts]
Thanks to everyone who was able to attend today, great discussion and some progress on both learning and development of the 2022 drivetrain. It is awesome that after one week we have both the PilotController and Drivetrain classes with an initial implementation.

I created a repository for the TicTacToe game - I encourage you to review it and even make modifications and improvements using what you’ve learned. [https://github.com/frc5567/2022TicTacToe](https://github.com/frc5567/2022TicTacToe)

The links that I would like everyone to please read are [classes](https://www.w3schools.com/java/java_classes.asp) and [methods](https://www.w3schools.com/java/java_methods.asp). These articles should reinforce what I’ve been trying to teach. Hoping that reading these provide the same information in a different medium to allow greater learning.

Finally, I don’t expect you all to be experts, but I do ask you keep trying to learn and grow your skills while we have time to go slowly. Later in the season we’re going to be under pressure to deliver and I want us to have the ability to do it!

**Comment by programmeruser517** [comment]
The repository link provided appears to lead to a 404 error.

**Comment by coverbeek** [comment]
It might be permissions - I’ll see if I can correct. The link does work for me, so it’s likely rights.

**Comment by coverbeek** [comment]
I hadn’t added the team to the access. Just added it - try again!
Thanks for the heads up!
